Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:3.9-alpine
+
+# Install prereqs
+RUN apk update 
+RUN apk add git gcc
+RUN apk add openssl-dev musl-dev openldap-dev postgresql-dev
+
+# Create directories
+RUN mkdir -p /var/log/fars
+RUN mkdir -p /opt/app/fars
+
+# Copy files
+COPY requirements.txt start-fars.sh /opt/app/
+RUN true
+COPY fars /opt/app/fars
+
+# Set workdir
+WORKDIR /opt/app
+
+RUN pip install --upgrade pip
+RUN pip install wheel
+
+# This is a hack that is required to install python-ldap, as alpine 
+# does not currentl support OpenLDAP 2.5+ which the package uses
+RUN echo -n "INPUT ( libldap.so )" > /usr/lib/libldap_r.so
+RUN pip install -r requirements.txt --cache-dir /opt/app/pip_cache
+
+# Create logfile
+RUN touch /var/log/fars/info.log
+RUN chmod -R 775 /var/log/fars/info.log
+RUN chmod -R 775 /opt/app/start-fars.sh
```

Comment:
You'll probably want to make sure these are mounted to the host for easy access, or a better solution might be to log to stdout so that you can view logs with `docker logs`

Reply:
The logs are currently linked to stdout with 
` RUN ln -sf /dev/stdout /var/log/fars/info.log` (line 30)